src/mlpack/core/data/binarize.hpp

```diff
@@ -45,10 +45,10 @@ void Binarize(const arma::Mat<T>& input,
   output.copy_size(input);
 
   const int totalElems = static_cast<int>(input.n_elem);
-  const T *inPtr = input.memptr();
-  T *outPtr = output.memptr();
+  const T* inPtr = input.memptr();
+  T* outPtr = output.memptr();
 
-  #pragma omp parallel for
+#pragma omp parallel for
```

Comment:
I think it's fine to indent the preprocessor macros, I don't think they should be always at 0 indentation.